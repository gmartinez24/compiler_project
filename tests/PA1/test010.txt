// Alex Garza
// Testing sequential punctuation marks with a mixed float to differentiate between period and float

main

;.,:()[]{}2.22.