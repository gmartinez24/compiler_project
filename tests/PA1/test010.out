MAIN
SEMICOLON
PERIOD
COMMA
COLON
OPEN_PAREN
CLOSE_PAREN
OPEN_BRACKET
CLOSE_BRACKET
OPEN_BRACE
CLOSE_BRACE
FLOAT_VAL	2.22
PERIOD
EOF
